src/LessMsi.Gui/AboutBox.cs

```diff
@@ -96,15 +99,48 @@ public string AssemblyCompany
 				}
 				return ((AssemblyCompanyAttribute)attributes[0]).Company;
 			}
-		}
+		}
 		#endregion
-
```

Comment:
@mega5800 I'd rather we just not translate the rtf file. Lets just leave it English. This complexity is not worth it.

Reply:
In this pull request, you are able to see a brand new logic I created for improving how our GUI app handles RTF files across different locales.

Instead of relying on a hardcoded RTF file in English, I've developed a generalized RTF template system. This template contains locale-independent content, such as names and links, while using placeholders for phrases that vary based on the user's language. 
My logic dynamically fills these placeholders with the appropriate locale-specific phrases, as defined in our strings file.

This approach not only streamlines our localization process but also ensures that all users, regardless of their locale, experience a polished and consistent interface. I believe this enhancement will contribute to a more personalized and professional user experience.

Reply:
I saw the code. That is actually what concerns me. I'd rather not have that added logic/complexity for the project to maintain for the content there that I do not think is important. The goal of the translation is to make it usable in other languages, and the about text - especially the few strings that are translatable there isn't important enough to translate to justify that added complexity (for example, it needs tests to prevent regressions, translations need to concern with RTF-ecoded content, their is new file I/O logic that needs to consider permissions, and odd paths, etc. etc.).

Reply:
I understand your point and will revert this change.